qark/plugins/task_affinity.py

```diff
@@ -0,0 +1,73 @@
+import sys, os, re
+
+sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
+
+from yapsy.IPlugin import IPlugin
+from plugins import PluginUtil
+from modules import common
+from lib.pubsub import pub
+import lib.plyj.parser as plyj
+
+class TaskAffinityPlugin(IPlugin):
+
+    NEW_TASK = r'FLAG\_ACTIVITY\_NEW\_TASK'
+    MULTIPLE_TASK = r'FLAG\_ACTIVITY\_MULTIPLE\_TASK'
+
+    def __init__(self):
+        self.name = 'Task Hijacking'
+
+    def target(self, queue):
+        files = common.java_files
+        global parser, tree, fileName
+        parser = plyj.Parser()
+        tree = ''
+        res = []
+        count = 0
+        for f in files:
+            count += 1
+            pub.sendMessage('progress', bar=self.name, percent=round(count * 100 / len(files)))
+            fileName = str(f)
+            try:
+                tree = parser.parse_file(f)
+            except Exception:
+                continue
+
+            try:
+                for import_decl in tree.import_declarations:
+                    if 'Intent' in import_decl.name.value:
+                        fileBody = str(open(fileName, 'r').read())
```

Comment:
Change to `file_body`

There are some problems with the current codebase using camelCasing but I think we should avoid using it unless we have to interact with that functionality. Python standard is snake_case.

This also might have some problems when opening the file. Consider changing this to a `with` for better exception handling and file closing. See [here](https://docs.python.org/2/tutorial/inputoutput.html#methods-of-file-objects)

```python
with open(fileName, 'r') as f:
    file_body = f.read()

if PluginUtil ...
```